### ISSUE TITLE
请问如何导出成onnx格式，我在转换时遇到了如下问题

### ISSUE THREAD

**Issue by lightandblue** [SpeechColab/GigaSpeech2#11]
2024-12-27 21:02:00,294 INFO [export-onnx.py:560] Exporting encoder                                                                                                                                                                          /home/node22_tmpdata/cywang/workspace/icefall/egs/gigaspeech2/ASR/zipformer/scaling.py:1486: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.                                                                                 zero = torch.tensor(0.0, dtype=x.dtype, device=x.device)                                                                                                                                                                                   /home/node22_tmpdata/cywang/workspace/icefall/egs/gigaspeech2/ASR/zipformer/scaling.py:1412: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.                                                                                 zero = torch.tensor(0.0, dtype=x.dtype, device=x.device)                                                                                                                                                                                   /home/node22_tmpdata/cywang/workspace/icefall/egs/gigaspeech2/ASR/zipformer/scaling.py:473: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!                                                                                                                        assert x.shape[self.channel_dim] == self.num_channels                                                                                                                                                                                      /home/node22_tmpdata/cywang/workspace/icefall/egs/gigaspeech2/ASR/zipformer/subsampling.py:330: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!                                                                                                                     assert x.size(1) == x_lens.max().item(), (x.size(1), x_lens.max())                                                                                                                                                                         /home/node22_tmpdata/cywang/workspace/icefall/egs/gigaspeech2/ASR/zipformer/subsampling.py:330: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert x.size(1) == x_lens.max().item(), (x.size(1), x_lens.max())
/home/work_nfs16/cywang/workspace/icefall/icefall/utils.py:1356: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated 
as a constant in the future. This means that the trace might not generalize to other inputs!
  max_len = max(max_len, lengths.max())
  
  
  我的脚本如下：
  ./zipformer/export-onnx.py \
    --epoch 17 \
    --avg  6 \
    --tokens data/lang_bpe_2000/tokens.txt \
    --exp-dir zipformer/exp_stage02_new_1 \
    --num-encoder-layers  2,2,4,5,4,2 \
    --downsampling-factor 1,2,4,8,4,2 \
    --feedforward-dim  512,768,1536,2048,1536,768 \
    --num-heads 4,4,4,8,4,4 \
    --encoder-dim 192,256,512,768,512,256 \
    --encoder-unmasked-dim  192,192,256,320,256,192 \
    --query-head-dim 32 \
    --value-head-dim 12 \
    --pos-head-dim  4 \
    --pos-dim 48 \
    --cnn-module-kernel  31,31,15,15,15,31 \
    --decoder-dim 512 \
    --joiner-dim 512 \
    --use-averaged-model False \
    --causal False \
    --chunk-size "16,32,64,-1" \
    --left-context-frames "64,128,256,-1"

**Comment by yfyeung** [comment]
留意到已经关闭了，如果后续有什么问题，欢迎随时联系
